src/components/filters/CategoryFilter.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Category } from '@graasp/sdk';
+
+import { useCategoriesTranslation } from '../../config/i18n';
+import {
+  buildSearchFilterCategoryId,
+  buildSearchFilterPopperButtonId,
+} from '../../config/selectors';
+import { Filter } from './Filter';
+
+type FilterProps = {
+  category: string;
+  title: string;
+  options?: Category[];
+  // IDs of selected options.
+  selectedOptions: string[];
+  onOptionChange: (key: string, newValue: boolean) => void;
+  onClearOptions: () => void;
+  isLoading: boolean;
+};
+
+// eslint-disable-next-line react/function-component-definition
```

Comment:
same here? or should we remove this eslint rule?

Reply:
For now I will let it, otherwise I will need to change all other components. I will create an issue.